File: C# Basics/Conditions/GodzillaVsKong.cs
using System;

namespace GodzillaVsKong
{
    class Program
    {
        static void Main(string[] args)
        {
            double budget = double.Parse(Console.ReadLine());
            int mutes = int.Parse(Console.ReadLine());
            double clothesPrice = double.Parse(Console.ReadLine());

            double decorePrice = budget * 10.0 / 100;
            double mutesExpense = mutes * clothesPrice;

            if (mutes > 150)
            {
                mutesExpense = mutesExpense * (1 - 10.0 / 100);
            }

            double totalExpense = decorePrice + mutesExpense;
            double finalBudget = budget - totalExpense;

            if (totalExpense > budget)
            {
                Console.WriteLine($"Not enough money!\nWingard needs {Math.Abs(finalBudget):f2} leva more.");
            }
            else
            {
                Console.WriteLine($"Action!\nWingard starts filming with {finalBudget:f2} leva left.");
            }
        }
    }
}

File: Java Basics/ConditionsEx/GodzillaVsKong.java
package ConditionsEx;

import java.util.Scanner;

public class GodzillaVsKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double movieBudget = Double.parseDouble(scanner.nextLine());
        int actors = Integer.parseInt(scanner.nextLine());
        double clothPrice = Double.parseDouble(scanner.nextLine());

        double decorPrice = movieBudget * (10 / 100d);
        if (actors > 150) {
            clothPrice *= (1 - 10 / 100d);
        }

        double totalCost = clothPrice * actors + decorPrice;

        if (totalCost > movieBudget) {
            System.out.printf("Not enough money!\nWingard needs %.2f leva more.", totalCost - movieBudget);
        } else {
            System.out.printf("Action!\nWingard starts filming with %.2f leva left.", movieBudget - totalCost);
        }
    }
}
